src/content/translations/sl/eth2/beacon-chain/index.md

```diff
@@ -4,7 +4,7 @@ description: Preberite več o oddajniški verigi – prvi večji nadgradnji Eth2
 lang: sl
 template: eth2
 sidebar: true
-image: ../../../../../assets/eth2/core.png
+image: ../../../assets/eth2/core.png
```

Comment:
@wackerow this is working in the deploy preview but I believe this is incorrect?